descriptors/ModuleDescriptor-template.json

```diff
@@ -151,6 +151,19 @@
           "modulePermissions" : [
             "permissions.users.item.get"
           ]
+        },
+        {
+          "methods" : [ "GET" ],
+          "pathPattern" : "/users-keycloak/auth-users/{userId}",
+          "permissionsRequired" : ["users-keycloak.auth-users.item.get"]
+        },
+        {
+          "methods" : [ "POST" ],
+          "pathPattern" : "/users-keycloak/auth-users/{userId}",
+          "permissionsRequired" : ["users-keycloak.auth-users.item.post"],
+          "modulePermissions" : [
+            "permissions.users.item.get"
```

Comment:
Each endpoint should be protected by unique permission. Please add new permissions and use them in the new endpoints